lib/libutee/include/sdp_pta.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2017, ARM Limited, All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Format
https://github.com/OP-TEE/optee_os/blob/master/documentation/copyright_and_license_headers.rst

Reply:
Ok, I will fix it.